eth/ethconfig/config.go

```diff
@@ -210,6 +211,9 @@ type Config struct {
 
 	// Check circuit capacity in block validator
 	CheckCircuitCapacity bool
+
+	// Max block range for eth_getLogs api method
+	MaxBlockRange int64
```

Comment:
Add a CLI param, e.g. `--rpc.getlogs.maxrange` (see how `CheckCircuitCapacity is set).